### PR TITLE
Partial LTO

### PR DESCRIPTION
**Pull request by goldenx86**
To save on RAM, and even improve performance a little bit, let's enable LTO only in the 2 critical projects that need it, core and video_core.

### CODE REVIEW

**Comment by FernandoS27** [comment]
Squash those commits. You are over commiting for a few miserable lines.

**Review by FernandoS27 — CHANGES_REQUESTED** [review]
Squash the commits and rebase like a normal human being (don't merge master, use git rebase for god's sake)

**Review by FernandoS27 — APPROVED** [review]

**Comment by goldenx86** [comment]
> Squash the commits and rebase like a normal human being (don't merge master, use git rebase for god's sake)

Ur ugly.

**Comment by Moonlacer** [comment]
> > Squash the commits and rebase like a normal human being (don't merge master, use git rebase for god's sake)
> 
> Ur ugly.

I second this motion

File: .ci/templates/build-msvc.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ parameters:
 steps:
 - script: choco install vulkan-sdk
   displayName: 'Install vulkan-sdk'
-- script: refreshenv && mkdir build && cd build && cmake -E env CXXFLAGS="/Gw /GA /Gr /Ob2" cmake -G "Visual Studio 17 2022" -A x64 -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON -DCMAKE_POLICY_DEFAULT_CMP0069=NEW -DYUZU_USE_BUNDLED_QT=1 -DYUZU_USE_BUNDLED_SDL2=1 -DYUZU_USE_QT_WEB_ENGINE=ON -DENABLE_COMPATIBILITY_LIST_DOWNLOAD=ON -DYUZU_ENABLE_COMPATIBILITY_REPORTING=${COMPAT} -DYUZU_TESTS=OFF -DUSE_DISCORD_PRESENCE=ON -DENABLE_QT_TRANSLATION=ON -DDISPLAY_VERSION=${{ parameters['version'] }} -DCMAKE_BUILD_TYPE=Release -DYUZU_CRASH_DUMPS=ON .. && cd ..
+- script: refreshenv && mkdir build && cd build && cmake -E env CXXFLAGS="/Gw /GA /Gr /Ob2" cmake -G "Visual Studio 17 2022" -A x64 -DCMAKE_POLICY_DEFAULT_CMP0069=NEW -DENABLE_LTO=ON -DYUZU_USE_BUNDLED_QT=1 -DYUZU_USE_BUNDLED_SDL2=1 -DYUZU_USE_QT_WEB_ENGINE=ON -DENABLE_COMPATIBILITY_LIST_DOWNLOAD=ON -DYUZU_ENABLE_COMPATIBILITY_REPORTING=${COMPAT} -DYUZU_TESTS=OFF -DUSE_DISCORD_PRESENCE=ON -DENABLE_QT_TRANSLATION=ON -DDISPLAY_VERSION=${{ parameters['version'] }} -DCMAKE_BUILD_TYPE=Release -DYUZU_CRASH_DUMPS=ON .. && cd ..
```
Comment (jbeich) [inline .ci/templates/build-msvc.yml:12]: Still uses `ENABLE_LTO` after it was renamed to `YUZU_ENABLE_LTO`

**Pull request merged**